File: fizzbuzz.fs
#! /usr/bin/gforth

\ Language:       Forth
\ Web site:       http://www.forth.org/
\                 https://www.gnu.org/software/gforth/
\ Last tested on: Ubuntu 17.04
\ Requires:       apt-get install gforth

: fizzbuzz ( -- )
    1
    101 1 ?do
        dup 15 mod 0= if
            ." FizzBuzz" cr
        else
            dup 3 mod 0= if
                ." Fizz" cr
            else
                dup 5 mod 0= if
                    ." Buzz" cr
                else
                    dup 1 .r cr
                then
            then
        then
        1+
    loop
    drop
    ;
fizzbuzz
bye

File: fizzbuzz.r
#!/usr/bin/R --slave -f

# Language:       R
# Web site:       https://www.r-project.org/
# Last tested on: Ubuntu 22.04.3
# Requires:       apt-get install r-base-core


for (i in 1:100) {
    if (i %% 15 == 0) {
        cat("FizzBuzz\n")
    }
    else if (i %% 3 == 0) {
        cat("Fizz\n")
    }
    else if (i %% 5 == 0) {
        cat("Buzz\n")
    }
    else {
        cat(i, "\n", sep="")
    }
}
